editor/src/styles/_login.scss

```diff
@@ -34,19 +37,61 @@ berta-login {
       transition: background-color .2s;
 
       &.facebook {
+        display: flex;
+        align-items: center;
+        padding: 10px 10px 10px 10px;
         background-color: $facebook-color;
 
+        & svg {
+          width: 25px;
+          padding-right: 20px;
+        }
+
+        & p {
+          margin-top: 0;
+          font-size: 18px;
+        }
+
         &:hover {
           background-color: darken($facebook-color, 10%);
         }
       }
 
-      &.google {
-        background-color: $google-color;
+      &.google:enabled {
```

Comment:
Why do you have `:enabled` state? The normal state is enabled - that's the default one.